e2e/bread-crumb.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { test, expect } from '@playwright/test';
+
+test.beforeEach(async ({ page }) => {
+  await page.goto(
+    '/learn/foundational-c-sharp-with-microsoft/write-your-first-code-using-c-sharp/trophy-write-your-first-code-using-c-sharp'
+  );
+});
+
+test.afterEach(async ({ page }) => {
+  await page.close();
+});
+
+test.describe('Challenge Breadcrumb Component Tests', () => {
+  test('breadcrumb nav links should be visible', async ({ page }) => {
```

Comment:
```suggestion
  test('breadcrumb nav links should be visible', async ({ page }) => {
    const leftBreadcrumb = page.getByTestId('breadcrumb-left');
    const rightBreadcrumb = page.getByTestId('breadcrumb-right');
```

Reply:
Hello Ahmad, are you suggesting to remove `const` and user `page` directly in `expect`?